classes/local/v1p2/csv_client_const_helper_two.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
This file name is pretty weird. Its namespaced down to v1p2 already, so putting two in the name should be redundant